lib/virt_autotest/utils.pm

```diff
@@ -108,7 +108,7 @@ sub is_xen_host {
 #check host to make sure it works well
 #welcome everybody to extend this function
 sub check_host {
-
+    record_soft_failure("Core dump in host fg. bsc#1204727") if script_run("journalctl | grep 'Started Process Core Dump'") == 0;
```

Comment:
This check for coredump should be useful in any feature test. Would you please also add this into virt_feature_test_base.pm at proper place so that classes inheriting this won't need to explicitly call it?

Reply:
Going to... it need take some time.

Reply:
I am also going to add 'call trace' check in bsc#1204662, althought test might fail ahead of the check.

Reply:
I guess this PR is planned to be merged without the two changes, right?

Reply:
They will be included in this PR. Wait one or two more days for updating and verifying.

Reply:
codes updated and two verification runs are updated.

[coredump - gi-guest_sles15sp4-on-host_developing-kvm](https://openqa.suse.de/tests/9831759)
[call trace + coredump - gi-guest_developing-on-host_developing-xen](https://openqa.suse.de/tests/9832054)